store/postwoman.js

```diff
@@ -102,6 +109,63 @@ export const mutations = {
     settings[key] = value;
   },
 
+  removeVariables({ editingEnvironment }, value) {
+    editingEnvironment.variables = value;
+  },
+
+  setEditingEnvironment(state, value) {
+    state.editingEnvironment = { ...value };
+  },
+
+  setVariableKey({ editingEnvironment }, { index, value }) {
+    editingEnvironment.variables[index].key = value;
+  },
+
+  setVariableValue({ editingEnvironment }, { index, value }) {
```

Comment:
If at all possible, could we not wrap the variable in quotes unless it is in fact just a string?
Perhaps a way of testing this would be with JSON.parse and a try/catch.

```typescript
// Example 1:
// Returns "test"
testValue("test");

// Example 2:
// Returns 300 (as opposed to "300")
testValue("300");

// Example 3:
// Returns true (as opposed to "true")
testValue("true");

function testValue(myValue: string){
    try {
        return JSON.parse(myValue);
    } catch(ex) {
        // Now we know it's a string just leave it as a string value.
        return myValue;
    }
}
```

Reply:
pretty much just copy pasta'd this function to the postwoman store and set it up for the setVariableValue mutation